psydac/api/postprocessing.py

```diff
@@ -187,8 +186,8 @@ def __init__(self, filename_space, filename_fields, comm=None, mode='w', save_mp
         self._space_info = {}
         self._spaces = []
 
-        if filename_space[-4:] != ".yml" and filename_space[-4:] != ".yaml":
-            filename_space += ".yml"
+        if not filename_space.split('.')[-1] in ['yml', 'yaml']:
```

Comment:
Can you not use `os.path.splitext`? See the [docs](https://docs.python.org/3/library/os.path.html#os.path.splitext)

Using`pathlib` is also possible, for a more object-oriented approach:
```python
from pathlib import Path
...

if Path(filename_space).suffix not in ('.yml', '.yaml'):
    ...
```

Reply:
Used `os.path.splitext` in [fba77de](https://github.com/pyccel/psydac/pull/234/commits/fba77de0bc83157d27c5f5bbf7f3fd75a05dc210)